### PR TITLE
Add QR code

### PR DESCRIPTION
**Pull request by tadatuta**
WIP!!!

Patzan said — patzan did ;)

### CODE REVIEW

**Comment by gela-d** [comment]
:) good PR!
Let's port it into `leveldb` tree?

**Comment by tadatuta** [comment]
@gela-d [done](https://github.com/bem/bem-site-engine/pull/81)

**Comment by gela-d** [comment]
Thanks for another PR, but we have some problem in this one.
Url which genereted is like`..text=/__example/bem-bl/dev/desktop.examples/b-icon/10-b-icon-src/10-b-icon-src.htm``, it's mean that user couldn't open this url.

File: src/blocks/common.blocks/block-example/block-example.bemtree [hunk 1]
```diff
@@ -74,6 +74,20 @@ block('block-example')(
...
+                        block: 'dropdown',
+                        mods: { switcher : 'link', theme: 'normal' },
+                        mix: [
+                            { block: 'block-example', elem: 'qr' },
+                            { block: 'block-example', elem: 'link' }
+                        ],
+                        switcher: 'QR',
+                        popup: {
+                            block: 'image',
+                            url: 'https://disk.yandex.net/qr/?format=svg&text=' + 'http://' + req.headers.host + url,
```
Comment (tadatuta) [inline src/blocks/common.blocks/block-example/block-example.bemtree:86]: it'd be better to get protocol from server context but I haven't found it.
think we can go with this implementation for now?

Comment (gela-d) [inline src/blocks/common.blocks/block-example/block-example.bemtree:86]: @tadatuta yes, sure.
Thank u!

**Pull request merged**